### ISSUE TITLE
1.18.2 Forge 40.1.59 The game crashed whilst initializing game

### ISSUE THREAD

**Issue by Billetiado10** [Engine-Room/Flywheel#149]
### Describe the Bug

The game crashed whilst initializing game
Error: java.lang.NullPointerException: Cannot invoke "java.io.File.getName()" because "this.f_10203_" is null
Código de error: -1

### Reproduction Steps

1. Download create-1.18.2-0.5.0.d
2. Download dependency flywheel-forge-1.18.2-0.6.5
3. Start de launcher Premium with dedicated 9GB RAM
4. I have another 85 more mods in my modpack

### Expected Result

Run a total of 87 mods alongside Oculus and Rubidium.

### Screenshots and Videos

VIDEO: https://1drv.ms/v/s!AgP8BvgSMFtCgkS7obElHYYUcMjU?e=NYA5yV

### Crash Report or Log

https://1drv.ms/t/s!AgP8BvgSMFtCgkOUCtnkdvu1Qkd8?e=GOKeaF

### Operating System

Windows 10 21h2 (compilation SO 19044.1889)

### What is your GPU?

Nvidia GeForce GTX 1050 ti 4VRAM

### Mod Version

0.6.5

### Minecraft Version

1.18.1

### Loader Version

Forge 40.1.59

### Other Mods

selene-1.18.2-1.17.9
FarmersDelight-1.18.2-1.1.2
create-1.18.2-0.5.0.d
NethersDelight-1.18.2-2.2.0
supplementaries-1.18.2-1.4.8
Patchouli-1.18.2-71.1
EarthMobs-1.18.2-1.1.2
DungeonsArise-1.18.2-2.1.50d-beta
curios-forge-1.18.2-5.0.7.1
blueprint-1.18.2-5.4.3
buzzier_bees-1.18.2-4.0.0
village-employment-1.18.2-1.5.1
villagertools-1.18-1.0.2
Aquaculture-1.18.2-2.3.7
stalwart-dungeons-1.18.2-1.2.2
sophisticatedcore-1.18.2-0.5.1.98
sophisticatedbackpacks-1.18.2-3.18.17.680
PrettyPipes-1.12.6
IBEEditor-1.18.2-2.1.1-forge
chipped-1.18.2-1.2.1-forge
CreativeCore_FORGE_v2.6.12_mc1.18.2
PlayerRevive_FORGE_v2.0.1_mc1.18.2
infernalmobs-1.18.5
citadel-1.11.3-1.18.2
Xaeros_Minimap_22.13.0_Forge_1.18.2
XaerosWorldMap_1.26.2_Forge_1.18.2
More+Babies-forge-1.18.1-0.5.2
notenoughanimations-forge-1.6.0-mc1.18.2
walljump-forge-1.18.1-1.3.7
BetterThirdPerson-Forge-1.18.2-1.8.1
MorePlayerModels-1.18.2.20220801
Bookshelf-Forge-1.18.2-13.2.47
Tips-Forge-1.18.2-5.0.3
auudio_forge_1.0.3_MC_1.18-1.18.2
drippyloadingscreen_forge_1.6.4_MC_1.18-1.18.2
corpse-1.18.2-1.0.1
architectury-4.9.83-forge
minecraft-comes-alive-7.3.16+1.18.2-forge
flashnpcs-1.18.1-1.1.4v2
fancymenu_forge_2.12.1-1_MC_1.18.2
loadmyresources_1.0.1_MC_1.18
konkrete_forge_1.5.1_MC_1.18-1.18.2
supermartijn642corelib-1.0.19-forge-mc1.18
supermartijn642configlib-1.1.6-forge-mc1.18
rechiseled-1.0.10a-mc1.18
JEITweaker-1.18.2-3.0.0.8
collective-1.18.2-4.37
CMDCam_FORGE_v2.0.5_mc1.18.2
Clumps-forge-1.18.2-8.0.0+15
configured-1.5.3-1.18.2
catalogue-1.6.2-1.18.2
appleskin-forge-mc1.18.2-2.4.1
AmbientSounds_FORGE_v5.0.14_mc1.18.2
creeperoverhaul-1.18.2-1.3.0-forge
Jade-1.18.2-forge-5.2.3
dynamiclightsreforged-1.18.2_v1.3.3
TerraBlender-forge-1.18.2-1.1.0.102
Oh_The_Biomes_You'll_Go-forge-1.18.2-1.4.2
JustEnoughResources-1.18.2-0.14.1.160
jei-1.18.2-9.7.1.232
rubidium-0.5.3a
oculus-mc1.18.2-1.2.5a
trashslot-forge-1.18.1-11.0.2
geckolib-forge-1.18-3.0.33
balm-3.2.0+0
[1.18.2]+SecurityCraft+v1.9.3
CraftTweaker-forge-1.18.2-9.1.185
modmenu-3.2.3
emotecraft-for-MC1.18.2-2.1.3-SNAPSHOT-build.30-forge
voicechat-forge-1.18.2-2.2.52
rubidium_extras-1.18.2_v1.3.1
PuzzlesLib-v3.3.5-1.18.2-Forge
VisualWorkbench-v3.3.0-1.18.2-Forge
CosmeticArmorReworked-1.18.2-v2
badpackets-forge-0.1.1
comforts-forge-1.18.2-5.0.0.4
FpsReducer2-forge-1.18.2-2.0
connectedglass-1.1.4-forge-mc1.18
CTM-1.18.2-1.1.4+4
craftingtweaks-forge-1.18.2-14.0.3
reap-1.18.2-1.0.0
emojiful-1.18.2-3.0.1
InventoryHud_[1.18].forge-3.4.4
elevatorid-1.18.2-1.8.4
carryon-1.18.2-1.17.0.8
alexsmobs-1.18.6

**Comment by f0kx** [comment]
same here

**Comment by last903347** [comment]
same problem here but I used 1.18.2, I once boot up successfully but after yesterday I updated botania, the game started to crash before initalizing and the log says the problem caused by flywheel

**Comment by Hahahavana** [comment]
The latest flywheel update seems to have broken a LOT of mods, even ones only tangentially related.

**Comment by hideki2k02** [comment]
Same here, the only solution i've found so far is to not update :(

**Comment by ccjjfdyqlhy** [comment]
So it's all because of flywheel?

**Comment by ccjjfdyqlhy** [comment]
No. Apparently not. I disabled flywheel and create but the problem still exists!

**Comment by ccjjfdyqlhy** [comment]
After testing, I figured out that it's because the version of OptiFine and Forge are not compatible. Try reinstall your Minecraft or OptiFine will help...

**Comment by rumrtco** [comment]
its the more player models try taking it out

**Comment by ccjjfdyqlhy** [comment]
Here's the thing:  
**There is no tolerance for [WARN] or [ERROR] events in this version of Forge.**  
Every modpack on CurseForge was using a modified version of it.  
I'm now finding a link for download.
